box/box.py

```diff
@@ -470,11 +479,13 @@ def __get_default(self, item, attr=False):
                 super().__setitem__(item, value)
         return value
 
-    def __box_config(self) -> Dict:
+    def __box_config(self, extra_namespace: Optional[str] = None) -> Dict:
         out = {}
         for k, v in self._box_config.copy().items():
             if not k.startswith("__"):
                 out[k] = v
+        if extra_namespace is not None:
```

Comment:
Need to change this to use a unique pre-set object like we do with `NO_DEFAULT` otherwise anything with `None` as a key will lose its namespace